packages/theme/src/theme/_forms.switch.scss

```diff
@@ -95,4 +90,8 @@ $switch-track-shadow-disabled: inset 2px 2px 7px 0 rgba(0, 0, 0, 0.2);
 			}
 		}
 	}
+	span {
```

Comment:
It's not necessary  a span?

Reply:
I didn't add a class on this, that's why I added an element selector but you're right, it's not a good selector.